docs/UserGuide.md

```diff
@@ -81,7 +82,7 @@ interactions recorded faster and simpler than traditional GUI applications.
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
 * If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `n/KFC n/MacDonalds`, only `n/MacDonalds` will be taken.
+  e.g. if you specify `n/Alice n/Bob`, only `n/Bob` will be taken.
```

Comment:
This is good and important, nice that its there 👍

Reply:
thank you!